src/wp-includes/load.php

```diff
@@ -753,7 +753,7 @@ function wp_start_object_cache() {
 			)
 		);
 
-		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins' ) );
+		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins', 'theme_json' ) );
```

Comment:
There are other places where `wp_cache_add_non_persistent_groups`.

https://github.com/WordPress/wordpress-develop/blob/ae42add90ef5b083368b87580fabbec1ada481f3/src/wp-includes/ms-blogs.php#L578

All places that currently have `array( 'counts', 'plugins' )`should also be updated with `, 'theme_json'`

Reply:
Done at https://github.com/WordPress/wordpress-develop/pull/3712/commits/27c8ab292fab4f7a9bb00fd0a972b385028f41b5